Makefile

```diff
@@ -44,12 +50,34 @@ $(call add-bindata,assets,./bindata/...,bindata,assets,pkg/operator/assets/binda
 $(call build-image,cert-manager-operator,$(IMAGE_OPERATOR),./images/ci/Dockerfile,.)
 $(call build-image,cert-manager-operator-bundle,$(IMAGE_OPERATOR_BUNDLE),./bundle/bundle.Dockerfile,./bundle)
 
+# generate scorecard image targets
+.PHONY: build/custom-scorecard-tests
+build/custom-scorecard-tests:
+	go build -mod=vendor $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./test
+
+.PHONY: build-e2e-image
+build-e2e-image:
+	docker build -f ./images/custom-scorecard-tests/Dockerfile -t $(TESTS_IMAGE) ./
+
+.PHONY: push-e2e-image
+push-e2e-image: build-e2e-image
+	docker push $(TESTS_IMAGE)
+
+.PHONY: run-e2e-image
+run-e2e-image:
+	IMAGE=${TESTS_IMAGE} envsubst < bundle/tests/scorecard/base.yaml > bundle/tests/scorecard/config.yaml 
+	oc new-project $(TESTS_NAMESPACE) || oc project $(TESTS_NAMESPACE)
+	oc create sa $(TESTS_SA) -n $(TESTS_NAMESPACE) || oc get sa $(TESTS_SA) -n $(TESTS_NAMESPACE)
+	oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:$(TESTS_NAMESPACE):$(TESTS_SA)
+	operator-sdk scorecard ./bundle --namespace=$(TESTS_NAMESPACE) --wait-time=$(TESTS_WAIT_TIME) -o json || oc delete project $(TESTS_NAMESPACE)
+	oc delete project $(TESTS_NAMESPACE)
+
 # exclude e2e test from unit tests
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
 
-# re-use test-unit target for e2e tests
-test-e2e: test-e2e-wait-for-stable-state
-	$(MAKE) GO_TEST_PACKAGES=./test/e2e/... test-unit
+test-e2e:
+	go build -o ./e2e-tests test/main.go
+	./e2e-tests --bundle ./bundle all
```

Comment:
[./bundle](https://github.com/openshift/cert-manager-operator/tree/master/bundle) is originally used for the the product bundle to be released, is it good to mix the test bundle under the product bundle? Can we not put test bundle stuff under product bundle dir?